quadlekBot.go

```diff
@@ -64,10 +65,20 @@ func run(c *cli.Context) error {
 
 	err = bot.RegisterPlugin(spotify.Register())
 	if err != nil {
-		fmt.Printf("error registering random plugin: %s", err.Error())
+		fmt.Printf("error registering spotify plugin: %s", err.Error())
 		return nil
 	}
 
+	if c.IsSet("tvdb-key") {
+		tvdbKey := c.String("tvdb-key")
+
+		err = bot.RegisterPlugin(nextep.Register(tvdbKey))
+		if err != nil {
+			fmt.Printf("error registering nextep plugin: %s", err.Error())
+			return nil
```

Comment:
If you return here, it'll exit the `main()` function and exit the bot. Is that what you want?